pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -817,6 +857,21 @@ func runHandler(ctx *domainContext, key string, c <-chan Notify) {
 				}
 				closed = true
 			}
+		case _, ok := <-cpuChannel:
+			if ok {
+				sub := ctx.subDomainConfig
+				c, err := sub.Get(key)
+				if err != nil {
+					log.Errorf("runHandler no config for %s", key)
+					continue
+				}
+				config := c.(types.DomainConfig)
```

Comment:
Panics if assertion fails.

Reply:
Should it panic? In the code above we do not panic in the same situation...